active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,356 @@
+| Ellipsis | * Inactive Button, often reading "...", used between indexes and before or after the first/last index to indicate an accessible but omitted range of indexes. <br> * MaxBefore and MaxAfter properties can be used to set  how many indices appear between the current page and the ellipsis before/after it. <br> * Visibility of the first/last index can be disabled. <br> * Only visible when using button panel as the display mode.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760513-85277c80-fc79-11e8-926e-3453ea29b0a3.png) |
+| PrefixText | * Text displayed before the editable ComboBox indexing component. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760540-93759880-fc79-11e8-9c4f-3c3222b95a13.png) |
+| NumberOfPages | * When a total number of indices (N) is given, this suffix string will appear after the editable ComboBox indexing component and read "of N". Localization will put "N" where it should be in a given language. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760557-9a9ca680-fc79-11e8-943f-3425d993fdeb.png) |
+
+## API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```c++ 
+enum PagerDisplayMode
+{
+    Auto,
+    ComboBox,
+    NumberBox,
+    ButtonPanel,
+};
+
+
+enum ButtonVisibilityMode
+{
+    Auto,
+    AlwaysVisible,
+    HiddenOnLast,
+    None,
+};
+
+runtimeclass PagerControlPageChangedEventArgs
+{
+    Integer CurrentPage{get; }:
+}
+
+runtimeclass PagerControl
+{
+    PagerControl();
+
+    PagerDisplayMode Display;
+    
+    Integer NumberOfPages;
+    
+    ButtonVisibilityMode FirstButtonVisibility;
+    ButtonVisibilityMode PreviousButtonVisibility;
+    ButtonVisibilityMode NextButtonVisibility;
+    ButtonVisibilityMode LastButtonVisibility;
+
+    FirstButtonCommand="FirstButtonPressedEvent"
+    PreviousButtonCommand="PreviousButtonPressedEvent"
+    NextButtonCommand="NextButtonPressedEvent"
+    LastButtonCommand="LastButtonPressedEvent"
+    PagerInputCommand="PagerInputEvent"
+    
+    IconSource FirstButtonGlyph;
+    IconSource PreviousButtonGlyph;
+    IconSource NextButtonGlyph;
+    IconSource LastButtonGlyph;
+    
+    String FirstButtonText;
+    String PreviousButtonText;
+    String NextButtonText;
+    String LastButtonText;
+
+    Windows.UI.Xaml.Input.ICommand FirstButtonCommand;
+    Windows.UI.Xaml.Input.ICommand PreviousButtonCommand;
+    Windows.UI.Xaml.Input.ICommand NextButtonCommand;
+    Windows.UI.Xaml.Input.ICommand LastButtonCommand;
+    Windows.UI.Xaml.Input.ICommand PagerInputCommand;
+    
+    Windows.UI.Xaml.Style FirstButtonStyle;
+    Windows.UI.Xaml.Style PreviousButtonStyle;
+    Windows.UI.Xaml.Style NextButtonStyle;
+    Windows.UI.Xaml.Style LastButtonStyle;
+    
+    
+    Integer NumberOfIndicesShowing;
+    Boolean EllipsisEnabled;
+    Boolean EllipsisShowFirstAndLast;
```

Comment:
Not clear what these Ellipsis properties are?

Reply:
I think they are for the following ellipsis:

![image](https://user-images.githubusercontent.com/16122379/86832988-33cfb100-c099-11ea-8411-3e0f4e65cce5.png)

Reply:
Looks like there's a description in the API Notes now too. Are these properties ignored when it's a ComboBox?

Reply:
Yes the ellipses properties are ignored when the developer chooses combo box or number box as the display mode. I can definitely try and find a better way to organize the properties for the different modes but I am not sure if there is a better way to generalize the properties without having subclasses or different classes for each of the three modes.